pentaho-aggdesigner-core/src/it/resources/log4j2.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ===================================================================== -->
+<!--                                                                       -->
+<!--  Log4j2 Configuration                                                  -->
+<!--                                                                       -->
+<!-- ===================================================================== -->
+
+<!--
+   | For more configuration information and examples see the Apache Log4j
+   | website: https://logging.apache.org/log4j/2.x/manual/configuration.html
+ -->
+
+<Configuration name="AggDesigner" status="INFO" dest="out">
+
+  <Appenders>
+
+    <!-- ================================= -->
+    <!-- Preserve messages in a local file -->
+    <!-- ================================= -->
+
+    <!-- A time/date based rolling appender -->
+    <RollingFile name="FILE" fileName="aggdesigner.log" filePattern="aggdesigner.log.%i" append="false" bufferedIO="true" immediateFlush="true">
+      <PatternLayout>
+        <Pattern>%d %-5p [%c] %m%n</Pattern>
+      </PatternLayout>
+      <Policies>
+        <OnStartupTriggeringPolicy />
+        <SizeBasedTriggeringPolicy size="500KB" />
+        <TimeBasedTriggeringPolicy />
```

Comment:
This line should be removed since the filePattern doesn't specify a date in there.